Source/inv.cpp

```diff
@@ -349,21 +349,20 @@ void DrawInvBelt(const CelOutputBuffer &out)
 
 /**
  * @brief Adds an item to a player's InvGrid array
- * @param playerNumber Player index

```

Comment:
Did you forget to replace the comment?

Reply:
If the description is just going to be "the player" i don't think it add any value and can just end up drifting from the actual code.

Reply:
Well... seems a bit arbitrary to selectively exclude parameters like this, but up to you I guess.